pandas/tests/extension/base/constructors.py

```diff
@@ -4,8 +4,10 @@
 import pandas.util.testing as tm
```

Comment:
ideally we *don't* / *shouldn't* import ``tm`` in any of the subclasses of EA for testing (otherwise easy to accidently NOT use ``self.assert_*``

Reply:
It's still used for things like `assert_raises_regex`. But maybe we could import them explicitly (`from pandas.util.testing import ...`